bin/needs-a-genny

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+genny_path = ->(slug) {
```

Comment:
since it seems that this file has chosen to elide parentheses where possible (according to lines 4 and 7), it seems that `(slug)` should become `slug` instead.

Reply:
And thank you for your suggestions.